### PR TITLE
fix: broken links in js docs

### PR DESCRIPTION
**Pull request by eswarclynn**

### CODE REVIEW

File: components/Header.tsx [hunk 1]
```diff
@@ -98,7 +110,7 @@ const Header: React.FC<Props> = ({ menuState, modal, setModal, docs, currentDocS
             <div className="head-right">
                 <div className="nav-links">
                     <button className={!isApiRef ? 'link-btn' : 'link-btn-active'} type="button">
-                        <Link href={`/${currentTech}/`}> Docs</Link>
+                        <Link href={`/${getCurrentTech() || 'javascript'}/`}> Docs</Link>
```
Comment (saikatmitra91) [inline components/Header.tsx:113]: why calling `getCurrentTech()` again ? 

can we reuse `currentTech` ?

File: components/Header.tsx [hunk 2]
```diff
@@ -50,25 +50,37 @@ const Header: React.FC<Props> = ({ menuState, modal, setModal, docs, currentDocS
...
+        }
+
+        return currentTech;
+    };
     // @ts-ignore
-    const currentTech =
-        // @ts-ignore
-        router.query.slug[0] === 'api-reference' ? router.query.slug[1] : router.query.slug[0];
     const routeAPIRef = () => {
+        const currentTech = getCurrentTech() || 'javascript';
```
Comment (saikatmitra91) [inline components/Header.tsx:69]: shouldnt this logic be inside `getCurrentTech` function ? 🤔

**Review by saikatmitra91 — CHANGES_REQUESTED** [review]

**Comment by saikatmitra91** [comment]
@eswarclynn can we also check if this page is needed or not 

https://www.100ms.live/docs/v2/client-sdks/Getting-Started-Javascript

**Comment by eswarclynn** [comment]
> @eswarclynn can we also check if this page is needed or not
> 
> https://www.100ms.live/docs/v2/client-sdks/Getting-Started-Javascript

No we're not, checked with the team too. Do you want it removed?

**Review by saikatmitra91 — APPROVED** [review]

**Pull request merged**